test/root_hooks.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+const blog = require('../blog_server');
+
+before((done) => {
+  blog.up().then(done);
+});
+
```

Comment:
I'm wondering if we should add some database cleanup/teardown around each test. Unfortunately, Mocha doesn't give us access to a given test's environment (or even the test to run), so something like [this](http://stackoverflow.com/questions/27220269/how-do-i-perform-an-around-or-aroundeach-with-mocha) might be the cleanest approach (and we could ditch that nasty `bind` if we used an arrow function, at least).